<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDemoWindow</class>
 <widget class="QWidget" name="SettingsDemoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingsDemoWindow</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Choices to Choose</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>39</y>
         <width>85</width>
         <height>104</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_A">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_B">
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_C">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_D">
          <property name="text">
           <string>D</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_E">
          <property name="text">
           <string>E</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>More Stuff!</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>261</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Only one...</string>
          </property>
          <widget class="QRadioButton" name="radioButton_ranknrmse">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>159</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Rank-nRMSE</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_nrmse">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>22</y>
             <width>159</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>nRMSE</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_rwnrmse">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>41</y>
             <width>159</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>RW-nRMSE</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_veryImportantNumber"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_paintbrush">
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_eraser">
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
